Makefile

```diff
@@ -1,18 +1,34 @@
-BUILD_NUMBER := 1
-CP_VERSION := 3.3.0-SNAPSHOT
+# You can override vars like REPOSITORY in a local.make file
+-include local.make
```

Comment:
That's a good question. We would put it all in the wrapper script.

As it is, though, it's still easy to do builds without the wrapper script. It might be nice to keep it that way.